cs3/app/provider/v1beta1/provider_api.proto

```diff
@@ -53,11 +53,38 @@ import "cs3/types/v1beta1/types.proto";
 // Any method MAY return UNKNOWN.
 // Any method MAY return UNAUTHENTICATED.
 service ProviderAPI {
-  // Returns the App provider URL
+  // Creates a new file to be opened by an app. The new file MAY be either
+  // empty or generated out of a template if the AppProvider has such capability.
+  rpc CreateFileForApp(CreateFileForAppRequest) returns (CreateFileForAppResponse);
```

Comment:
From the UI perspective. As a I user I should be able to create for example: "New MS Word document" or "New OnlyOffice document". I think we may need to add the `app` name in the payload.

Reply:
Just discussed with Lisa and Diogo: the creation of the file (either touch or copy from template) takes place before firing the app, and it does not involve the app at all - at least, so far we don't know any app (definitely not the WOPI ones) that expose an endpoint to _create_ a new file. The new file/template operation is therefore only on the storage and app independent.